dist/tools/travis-scripts/get-pkg-list.py

```diff
@@ -29,7 +29,7 @@
 known_mcu_groups = arm_mcu_groups + msp_mcu_groups + x86_mcu_groups + \
     avr8_mcu_groups + static_tests_groups
 
-common_pkgs = ["pcregrep", "libpcre3", "python3"]
+common_pkgs = ["pcregrep", "libpcre3", "python3", "python3-pexpect"]
```

Comment:
Also, #4911 determined that the Ubuntu version of `pexpect` for Python 2 has bugs / is incompatible with newer versions. Please see if this is true for the Python 3 version to and if yes use `pip3` rather then APT to install the module.